### ISSUE TITLE
Tests fail

### ISSUE THREAD

**Issue by mineo** [musicbrainz/picard-plugins#30]
```
======================================================================
ERROR: test_valid_json (test.GenerateTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/wieland/dev/picard-plugins/test.py", line 73, in test_valid_json
    self.assertIsInstance(data['api_version'], basestring)
KeyError: 'api_version'

----------------------------------------------------------------------
```

**Comment by zas** [comment]
Tests have to be rewritten so they use the same code as generate.py

**Comment by dufferzafar** [comment]
It's not the generate script actually. The problem is that the [keep plugin](https://github.com/musicbrainz/picard-plugins/blob/master/plugins/keep/keep.py) uses [two lines](https://github.com/musicbrainz/picard-plugins/blob/master/plugins/keep/keep.py#L9-L10) for PLUGIN_API_VERSIONS, so the generate script can't extract that field from the file. 

We could either update the generate script to handle such cases or just change keep.py to get the code into a single line.

**Comment by dufferzafar** [comment]
I think Travis should've warned us that the tests were failing before I merged pull #24.

**Comment by zas** [comment]
@dufferzafar : i modified the way we parse plugin metadata in https://github.com/musicbrainz/picard-plugins/pull/31
